src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -119,12 +120,18 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings)
             if (packageSources.Count < 1)
```

Comment:
It didn't look in config before, I don't know the real purpose of "this.Sources", if it should override nuget.config sources or not. So I would keep the existing code as-is unless there's a problem.

---
In reply to: [62935575](https://github.com/NuGet/NuGet.Client/pull/586#discussion_r62935575) [](ancestors = 62935575)